docs/how_to/pytorch_install/pytorch_install.md

```diff
@@ -48,107 +37,89 @@ Follow these steps:
+(install_pytorch_wheels)=
 
-#### Option 2: Install PyTorch Using Wheels Package
+## Using a wheels package
 
-PyTorch supports the ROCm platform by providing tested wheels packages. To
-access this feature, refer to
-[https://pytorch.org/get-started/locally/](https://pytorch.org/get-started/locally/)
-and choose the "ROCm" compute platform. {numref}`Installation-Matrix-from-Pytorch` is a matrix from <https://pytorch.org/> that illustrates the installation compatibility between ROCm and the PyTorch build.
+PyTorch supports the ROCm platform by providing tested wheels packages. To access this feature, go
+to [https://pytorch.org/get-started/locally/](https://pytorch.org/get-started/locally/). In the interactive
+table, choose ROCm from the _Compute Platform_ row.
 
-```{figure} ../../data/how_to/magma_install/image.006.png
-:name: Installation-Matrix-from-Pytorch
----
-align: center
----
-Installation Matrix from Pytorch
-```
+1. Choose one of the following three options:
 
-To install PyTorch using the wheels package, follow these installation steps:
+   **Option 1:**
 
-1. Choose one of the following options:
-   a. Obtain a base Docker image with the correct user-space ROCm version
-      installed from
-      [https://hub.docker.com/repository/docker/rocm/dev-ubuntu-20.04](https://hub.docker.com/repository/docker/rocm/dev-ubuntu-20.04).
+   Download a base Docker image with the correct user-space ROCm version. See:
+   [https://hub.docker.com/repository/docker/rocm/dev-ubuntu-20.04](https://hub.docker.com/repository/docker/rocm/dev-ubuntu-20.04).
 
-   or
+   **Option 2:**
 
-   b. Download a base OS Docker image and install ROCm following the
-      installation directions in the section
-      [Installation](../../deploy/linux/install.md). ROCm 5.2 is installed in
-      this example, as supported by the installation matrix from
-      <https://pytorch.org/>.
+   Download a base OS Docker image and install ROCm using the directions in the
+   [Installation](../../deploy/linux/install.md) section.
 
-   or
+   **Option 3:**
 
-   c. Install on bare metal. Skip to Step 3.
+   Install on bare metal. If using this method, skip step 2 (starting a Docker container).
 
-      ```bash
-      docker run -it --device=/dev/kfd --device=/dev/dri --group-add video rocm/dev-ubuntu-20.04:latest
-      ```
+   ```bash
+   docker run -it --device=/dev/kfd --device=/dev/dri --group-add video rocm/dev-ubuntu-20.04:latest
+   ```
 
-2. Start the Docker container, if not installing on bare metal.
+2. Start the Docker container (skip this step for bare metal installations).
 
    ```dockerfile
    docker run -it --device=/dev/kfd --device=/dev/dri --group-add video rocm/dev-ubuntu-20.04:latest
    ```
 
-3. Install any dependencies needed for installing the wheels package.
+3. Install the required dependencies for the wheels package.
 
    ```bash
    sudo apt update
    sudo apt install libjpeg-dev python3-dev
    pip3 install wheel setuptools
```

Comment:
there will be no `pip3` in some base images. So we need to append `python3-pip` package to `apt install` command
```bash
   sudo apt update
   sudo apt install libjpeg-dev python3-dev python3-pip
   pip3 install wheel setuptools
   ```